libs/api/domains/driving-license/src/lib/graphql/main.resolver.ts

```diff
@@ -27,13 +27,21 @@ const namespace = '@island.is/api/driving-license'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
 @Scopes(ApiScope.internal)
-@Resolver()
+@Resolver(QualityPhoto)
```

Comment:
Might want to create a new resolver for QualityPhoto fields.

Reply:
I  thought the syntax would be:
```typescript
@Resolver(() => QualityPhoto)
```

But yeah, agree with Eiki. Should have a new file specific for that resolver if it's resolving the GraphQL type's fields.